packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -102,7 +106,7 @@ export function withResponsiveMode<TProps extends { responsiveMode?: ResponsiveM
   return hoistStatics(ComposedComponent, resultClass);
 }
 
-function getResponsiveMode(currentWindow: Window | undefined): ResponsiveMode {
+export const getResponsiveMode = (currentWindow: Window | undefined): ResponsiveMode => {
```

Comment:
Please switch this back to `function` syntax too. Or if you must use lambdas, also switch `withResponsiveMode` so it's consistent within the file